### ISSUE TITLE
Manually change the quantity even if for single digit number

### ISSUE THREAD

**Issue by sanskar1419** [cn10xdev/ECommerce_Product_Page#22]
_On the web page we can increase or decrease the quantity by one, but when a single digit number is already there then we cannot completely upgrade to a new quantity_

For example, If product 2 has 4 quantities already in the cart and we want to change the quantities to 92 then we have to increase the quantity 5 times to make it 9 and then add 2. Is a lengthy process.

Instead of this, we could have added the feature of manually editing the quantity with the exception that quantities should not be less than or equal to 0. 

[E-Commerce-Product-Page-ezgif.com-mute-video.webm](https://github.com/user-attachments/assets/9a710c0a-ce61-45e9-9408-add7872c513a)

**Comment by Ezio-Sarthak** [comment]
Thanks for bringing this issue @sanskar1419 ! Yeah this is a bug that needs to be fixed. Let me know if you would like to work on the same

**Comment by sanskar1419** [comment]
Definitely, I would love to work on it. Will raise PR as soon fix the bug.
